homeworks/artur.kobyliatsky_Allocene/GamesProject/style/about.css

```diff
@@ -0,0 +1,233 @@
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+
+
+/*header section*/
+
+.header {
+  position: fixed;
+  width: 100%;
+  top: 0;
+  left: 0;
+  height: 70px;
+  background-color: #EF4934;
+}
+
+.header__box {
+  display: flex;
+  align-items: center;
+  height: 100%;
+}
+
+.header__box-content {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  margin: 0 auto;
+}
+
+.header__logo-section {
+  display: flex;
+  align-items: center;
+}
+
+.header__logo-section img {
+  cursor: pointer;
+}
+
+.header__menu-section {
+  display: flex;
+  align-items: center;
+}
+
+.header__menu-section a {
+  text-decoration: none;
+  color: #FFF;
+}
+
+.header__menu-section a:active {
+  transition: all .3s;
+  color: #FFF;
+  padding-bottom: 7px;
+  border-bottom: 1px solid #FFF;
+}
+
+.header__menu-section a:hover {
+  transition: all .3s;
+  color: #FFF;
+  padding-bottom: 7px;
+  border-bottom: 1px solid #FFF;
+}
+
+.menu__list {
+  display: flex;
+}
+
+.menu__list li {
+  list-style: none;
+  margin: 0 0 0 20px;
+}
+
+.menu__list-link {
+  font-size: 16px;
+  text-decoration: none;
+}
+
+
+/*main section*/
+
+main {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.contain {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 140px;
```

Comment:
Why `margin-top` is so big here and on the Games page?

Reply:
I just took into account the distance along with the search block, I already corrected it

Reply:
Because this distance is indicated in the layout, when changed, to be honest, it doesn’t look very good
Top
140px

Reply:
Ok, you have the absolute header (out of the flow). But its height is 70px. And by design the gap between h1 and top bars is 40px. 70+40=110. You have 140px and the visual difference is easy to catch.
You can fix it in the next task.
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/c2b58d15-b47a-492e-9854-9400fe0c0323)